javalin/src/test/java/io/javalin/TestJavalinVue.kt

```diff
@@ -162,4 +162,28 @@ class TestJavalinVue {
         assertThat(slot.captured).contains("""src="https://cdn.jsdelivr.net/webjars/""")
     }
 
+    @Test
+    fun `@inlineFile functionality works as expected if not-dev`() = TestUtil.test { app, http ->
+        val ctx = mockk<Context>(relaxed = true)
+        every { ctx.url() } returns "http://123.123.123.123:1234/"
+        VueComponent("<test-component></test-component>").handle(ctx)
+        val slot = slot<String>().also { verify { ctx.html(html = capture(it)) } }
+        assertThat(slot.captured).contains("""<script>let a = "Always included"</script>""")
+        assertThat(slot.captured).contains("""<script>let b = "Included if not dev"</script>""")
+        assertThat(slot.captured).doesNotContain("""<script>@inlineFileDev("/vue/scripts-dev.js")</script>""")
+        assertThat(slot.captured).doesNotContain("""<script>@inlineFile""")
+    }
+
+    @Test
+    fun `@inlineFile functionality works as expected if dev`() = TestUtil.test { app, http ->
+        val ctx = mockk<Context>(relaxed = true)
+        every { ctx.url() } returns "http://localhost:1234/"
+        VueComponent("<test-component></test-component>").handle(ctx)
+        val slot = slot<String>().also { verify { ctx.html(html = capture(it)) } }
+        assertThat(slot.captured).contains("""<script>let a = "Always included"</script>""")
+        assertThat(slot.captured).contains("""<script>let b = "Included if dev"</script>""")
```

Comment:
Maybe add the (last) check?
`assertThat(slot.captured).doesNotContain("""<script>let b = "Included if not dev"</script>""")`

Reply:
Will do !